### PR TITLE
Add explanation that groups are unofficial

### PR DESCRIPTION
**Pull request by QuincyLarson**

### CODE REVIEW

File: LocalGroups-List.md [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 (This map is very out of date and you should look for your city in the following list instead.  
 If you're up to it, please help us [edit the map](https://www.google.com/maps/d/edit?mid=zH0hxoM17_fk.kDBMlxfk0BZ8) and update the missing cities in there.)  
+
 If your city isn't listed here, you should [create your own Local Group](LocalGroups-Create).
+
+Please note that all local groups are unofficial, independently organized groups of Free Code Camp community members. They aren't aren't affiliated with the company that hosts freecodecamp.com, Free Code Camp, Inc., and the company doesn't control or endorse what local group members and organizers do on their own.
```
Comment (alayek) [inline LocalGroups-List.md]: > They aren't aren't 

You need to remove one _aren't_.

**Comment by alayek** [comment]
Other than What I have mentioned in inline comment,

# LGTM :+1:

File: LocalGroups-List.md (hunk 1 above)
Comment (theflametrooper) [inline LocalGroups-List.md]: ~~of Free Code Camp community members.~~
by Free Code Camp community members.

**Comment by abhisekp** [comment]
@alayek I've fixed the issues. This can be merged happily :smiley:

**Pull request merged**